doc/releases/release-notes-3.4.rst

```diff
@@ -11,24 +11,24 @@ Major enhancements with this release include:
 
 * Input subsystem: handles input events from various types of input devices and
   distribute them to other threads in the application.
-* Barrierr API: add architecture agnostic API for data memory barriers.
+* Barrier API: add architecture agnostic API for data memory barriers.
 * USB Device support:
 
-  * USB device controller API (UDC API)
+  * USB device controller API (UDC API).
   * USB device controller API and nRF USBD controller driver.
```

Comment:
this line looks like it's repeating part of the line above? should this be made just one line?